src/lib/tabs/tabRelics/RelicsTab.jsx

```diff
@@ -684,14 +717,36 @@ export default function RelicsTab() {
         setOpen={setEditModalOpen}
         open={editModalOpen}
       />
-      <Flex vertical gap={10}>
+      <Flex vertical gap={10} style={{ width: '100%' }}>
 
         <RelicFilterBar
           setValueColumns={setValueColumns}
           valueColumns={valueColumns}
           valueColumnOptions={valueColumnOptions}
         />
 
+        <Collapse 
+          size='small'
+          items={[
+            {
+              key: '1',
+              label: t('RelicGrid.RecentRelics', 'Recent Relics'),
```

Comment:
Thoughts on "Recently updated relics" instead? B/c lock/unlock/enhance also puts them in here